packaging/bootupd.spec

```diff
@@ -0,0 +1,56 @@
+%bcond_without check
+%global __cargo_skip_build 0
+
+%global crate bootupd
+
+Name:           %{crate}
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        Bootloader updater
+
+License:        ASL 2.0
+URL:            https://crates.io/crates/bootupd
+Source0:        https://crates.io/api/v1/crates/%{crate}/%{version}/download#/%{crate}-%{version}.crate
+Source1:        https://github.com/coreos/bootupd/releases/download/v%{version}/%{crate}-%{version}-vendor.tar.gz
+
+# For now, see upstream
+ExclusiveArch:  x86_64
+BuildRequires:  openssl-devel
+%if 0%{?rhel} && !0%{?eln}
+BuildRequires: rust-toolset
+%else
+BuildRequires: cargo
+BuildRequires: rust
+%endif
+BuildRequires:  systemd
+
+%description
+%{summary}
+
+%files
+%license LICENSE
+%doc README.md
+%{_libexecdir}/bootupd
+%{_unitdir}/*
+
+%prep
+# FIXME shouldn't both source0/source be extracted with this?
+%autosetup -n %{crate}-%{version} -p1
+%autosetup -n %{crate}-%{version} -p1 -a 1
+# https://github.com/rust-lang-nursery/error-chain/pull/289
+find -name '*.rs' -executable -exec chmod a-x {} \;
+mkdir -p .cargo
+cat >.cargo/config << 'EOF'
```

Comment:
I _think_ (but could be wrong :smile:) [`%cargo_prep`](https://pagure.io/fedora-rust/rust2rpm/blob/master/f/data/macros.cargo) can help out here and using it may [make it easier to get the package approved](https://docs.fedoraproject.org/en-US/packaging-guidelines/Rust/#_others). Looking at the macro it is writing the config out and setting things similarly.